modules/persistent-storage-local-create-cr.adoc

```diff
@@ -62,6 +62,7 @@ local volumes.
 <4> The file system that is created when the local volume is mounted for the
 first time.
 <5> The path containing a list of local storage devices to choose from.
+<6> Replace this value with your actual local disks filepath to the LocalVolume resource, such as `pod/local-disks-local-diskmaker-ldldw`. This will automatically create the PVs for these local disks.
```

Comment:
For the explanation, how about:
```
Replace this value with your actual local disks filepath to the LocalVolume resource, such as `/dev/xvdg`.  There are PVs created for these local disks when the provisioner is deployed successfully.

Reply:
That makes sense, @qinpingli. I've updated to reflect your suggestion. Thanks!